.github/workflows/dotnet-build.yml

```
@@ -1,16 +1,29 @@
-name: .NET Build
+﻿name: .NET Build
+
 on: [push]
 
 jobs:
   build:
-    name: Build
+
     runs-on: windows-latest
+
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Setup
-        uses: actions/setup-dotnet@v1
-      - name: Build
-        run: dotnet build --configuration Release
-      - name: Test
-        run: dotnet test
+    - uses: actions/checkout@v1
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
```

```suggestion
      uses: actions/setup-dotnet@v5
```

https://github.com/actions/setup-dotnet

感觉 8.x 和 9.x 是不用安装的，自带的就有了